build-config/make/sysroot.make

```diff
@@ -52,6 +55,10 @@ $(DEV_SYSROOT_INIT_STAMP): $(TREE_STAMP) | $(XTOOLS_BUILD_STAMP)
 	$(Q) echo "==== Preparing a new development sysroot ===="
 	$(Q) rm -rf $(DEV_SYSROOT)
 	$(Q) cp -a $$($(CROSSBIN)/$(CROSSPREFIX)gcc -print-sysroot) $(DEV_SYSROOT)
+	$(Q) rsync -rl $(XTOOLS_INSTALL_DIR)/$(TARGET)/$(TARGET)/lib $(DEV_SYSROOT)
+       ifeq ($(ARCH),$(filter $(ARCH),arm64 x86_64))
+		$(Q) rsync -rl $(XTOOLS_INSTALL_DIR)/$(TARGET)/$(TARGET)/lib64 $(DEV_SYSROOT)
+       endif
```

Comment:
For `accton_as5712_54x` the busybox phase is pausing for user input, which we can't have.

I'm getting this question:
```
monster-15:~/onie-cn/onie/build-config$ make -j48 MACHINEROOT=../machine/accton/ MACHINE=accton_as5712_54x busybox            
====  Building busybox-1.25.1 ====
make[1]: Entering directory `/work/monster-15/curt/trees/onie-cn/onie/build/accton_as5712_54x-r0/busybox/busybox-1.25.1'
scripts/kconfig/conf -s Config.in
# 
# using defaults found in .config
# 
* 
* Restart config...
* 
* 
* ONIE system EEPROM config
* 
Choose which device type of EEPROM
> 1. i2c (SYS_EEPROM_DEVICE_I2C)
  2. MTD (SYS_EEPROM_DEVICE_MTD)
  3. disk (not recommended) (SYS_EEPROM_DEVICE_DISK)
  4. sysfs file (SYS_EEPROM_SYSFS_FILE)
choice[1-4?]: 1
i2c bus (SYS_EEPROM_I2C_DEVICE) [/dev/i2c-1] /dev/i2c-1
device address on the i2c bus (SYS_EEPROM_I2C_ADDR) [0x57] 0x57
eeprom address bits (SYS_EEPROM_I2C_MEM_ADDR_BITS) [8] (NEW) 
```

That needs to get cleaned up.